tika/server.go

```diff
@@ -128,6 +128,10 @@ var command = exec.Command
 // Server. Start will wait for the server to be available or until ctx is
 // cancelled.
 func (s *Server) Start(ctx context.Context) error {
+	if _, err := os.Stat(s.jar); os.IsNotExist(err) {
+		return err
+	}
```

Comment:
Should we do this in `NewServer`? Could see doing it in either function.

Reply:
I agree, could be in either. Pathing of where things are called from could come into play, which is why I put it here and not in NewServer.

Reply:
Sounds good to me.